### DISCUSSION TITLE
Reference Data Set - Charles TOTH

### DISCUSSION THREAD

**Discussion by PhenolicPanther** [rodralez/NaveGo#155] [General]
Hi!

I was reading some of the papers related to NaveGo and I saw cited:
"Reference Data Set for Performance Evaluation of  MEMS-based Integrated Navigation Solutions "

I was wondering if anyone still had access to this data, I'm really struggling to find it online.

Thanks and awesome job!

**Comment by rodralez** [comment]
Hello,

Thank you for your kind words.

We have Dr. Charles Toth's permission to share this dataset. We can share it "as is" in the `examples/real-data` folder.

Give me a couple of days to see if the size of the files can be uploaded to Github.

Kind regards.

**Reply by PhenolicPanther** [reply]
Thank you very much it would be great to keep alive such data and not lose it.

I would suggest, also adding it to the repository readme, so that it would become searchable by search engines, providing visibility to the project and keeping alive Toth's past work.

If you need help with the file hosting feel free to ping me.

Best regards,

Omar

**Comment by rodralez** [comment]
Hi Omar,

You can download this dataset from:

[https://www.dropbox.com/sh/qx9waub1oo2qlvq/AAAnGAOSvvyFscSTUKyFkVvda?dl=0](https://www.dropbox.com/sh/qx9waub1oo2qlvq/AAAnGAOSvvyFscSTUKyFkVvda?dl=0)

Kind regards.

**Comment by rodralez** [comment]
Omar,

I was thinking...

Maybe you can create a NaveGo example using this dataset. It could be a great contribution to our tool.

If available, you should follow the NaveGo contribution workflow:

https://github.com/rodralez/NaveGo/blob/master/doc/navego-contribution-workflow.md

Please, let me know if you need any help to accomplish this task.

Kind regards.
